cmd/kubernetesDeploy.go

```diff
@@ -220,19 +249,19 @@ func runKubectlDeploy(config kubernetesDeployOptions, command command.ExecRunner
 
 		// first check if secret already exists
 		kubeCheckParams := append(kubeParams, "get", "secret", config.ContainerRegistrySecret)
-		if err := command.RunExecutable("kubectl", kubeCheckParams...); err != nil {
+		if err := utils.RunExecutable("kubectl", kubeCheckParams...); err != nil {
```

Comment:
This branch is only for `InNotFound` errors. This somehow hides all unexpected errors.